update/delta/delta_1.3.7_001_search_opened_valves.sql

```diff
@@ -0,0 +1,207 @@
+alter table qwat_network.network add column if not exists network_id serial ;
+
+/*
+Check if a node is a valve
+Is the parameter _check_if_network_function is true, then we also check if the valve is a network valve
+*/
+CREATE OR REPLACE FUNCTION qwat_network.ft_check_node_is_valve(_id integer, _check_if_network_function boolean)
```

Comment:
as far as I know, normally in QWAT, a valve shouldn't be on a node.
Is there any tolerance to this?